agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -24,7 +24,7 @@ def on_node_info(topic, info)
       @node_name = info['name']
       statsd_conf = info.dig('grid', 'stats', 'statsd')
       if statsd_conf
-        info "exporting stats via statsd to udp://#{statsd_conf['server']}:#{statsd_conf['port']}"
+        debug "exporting stats via statsd to udp://#{statsd_conf['server']}:#{statsd_conf['port']}"
```

Comment:
Imho this is valuable information.. and it is only logged when agent starts or when statsd configuration changes (does not spam logs).